packages/celotool/register.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Usage: register.sh [data dir] [geth dir]
+TYPE=${1:-"account"}
+DATA_DIR=${2:-"/tmp/e2e/group"}
+FAUCET=${3:-"0x47e172f6cfb6c7d01c1574fa3e2be7cc73269d95"}
+GETH_DIR=${4:-"/home/dimi/celo/celo-blockchain"}
```

Comment:
`dimi` should not be included. Please check for leaking your local environment when creating PRs.